components/src/dynamo/vllm/args.py

```
@@ -48,6 +49,7 @@ class Config(DynamoRuntimeConfig, DynamoVllmConfig):
     # mirror vLLM
     model: str
     served_model_name: Optional[str] = None
+    served_model_aliases: list[str] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)